src/providers/commandPalette.ts

```diff
@@ -65,14 +66,25 @@ export const useCommandPaletteStore = () => {
 
   const settleQuery = () => {
     commandPaletteStore.query = commandPaletteStore.currentInput
+    if (commandPaletteStore.currentInput) {
```

Comment:
```suggestion
    if (commandPaletteStore.currentInput !== '') {
```
空文字列でないときということを明示しておきたいです